src/app/components/add-new-category-item/add-new-category-item.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { AddNewCategoryItemComponent } from './add-new-category-item.component';
+
+describe('AddNewCategoryItemComponent', () => {
+  let component: AddNewCategoryItemComponent;
+  let fixture: ComponentFixture<AddNewCategoryItemComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [AddNewCategoryItemComponent]
+    })
+    .compileComponents();
+    
+    fixture = TestBed.createComponent(AddNewCategoryItemComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```

Comment:
add test for functionality, like - should add category to list, should dislpay valdiaton when not fill all required inputs etc.